doc/onboarding.md

```diff
@@ -32,33 +33,38 @@ onboarding session.
 ## Local setup
 
   * git:
-    * make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
-    * usually PR from your own github fork
+    * Make sure you have whitespace=fix: `git config --global --add apply.whitespace fix`
```

Comment:
I’ve been mentioning manually sometimes that you can set this locally if you really want to… I know *I* would like to have this as a global option, but sure, if others don’t, I can change this.